core/arch/arm/kernel/user_ta.c

```diff
@@ -490,7 +489,7 @@ static uint32_t user_ta_get_instance_id(struct tee_ta_ctx *ctx)
 	return to_user_ta_ctx(ctx)->vm_info->asid;
 }
 
-static const struct tee_ta_ops user_ta_ops __rodata_unpaged = {
+static const struct tee_ta_ops user_ta_ops = {
```

Comment:
Please keep the `__rodata_unpaged`. It's not supposed to affect the dependency tree, but it guarantees that  `user_ta_ops` itself doesn't end up in the paged part.

Reply:
ok